build/implementations/get-testcases-grouped-by-rule.js

```diff
@@ -0,0 +1,23 @@
+const axios = require('axios')
+const { config } = require('./../../package.json')
+
+/**
+ * Get testcases of rules
+ * - group them by `ruleId`
+ */
+const getTestcasesGroupedByRule = async () => {
+	const testcasesUrl = config['testcases-url']
```

Comment:
You've got a `testcases.json` file sitting in `./public/`. Why are you loading it from a URL? Wouldn't you just always have outdated results that way?

Reply:
This is the tricky one. `testcases.json` is generated during `gatsby-node`/ hooks run. 

But the implementations are generated prior to gatsby run. 

Need to shuffle testcase generation out of gatsby hook :/